apps/homescreen/js/pages.js

```diff
@@ -219,14 +219,11 @@
       // Remove child immediately, datastore operations can be quite slow
       this.removeCard(this.selectedCard);
 
-      this.pagesStore.get(id).then(entry => {
-        entry.data.pinned = false;
-        this.pagesStore.datastore.put(entry.data, id).then(() => {},
-          e => {
-            console.error('Error unpinning page:', e);
-          });
-      }, e => {
-        console.error('Error retrieving page to unpin:', e);
+      // TODO: fix LazyLoader mock to support promises
+      return new Promise((resolve, reject) => {
+        LazyLoader.load(['shared/js/places_model.js'], () => {
```

Comment:
The homescreen app can be the only one not to use the shared places model, but then we do need a code comment about that, so that if someone changes the code, they do keep into account compatibility with the schema used by all other apps that write to the `places` datastore (namely system, operatorvariant, and sync).